### PR TITLE
BUILD-2298 Fix promotion

### PR DESCRIPTION
**Pull request by tomverin**
Fixes #

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -41,11 +41,9 @@ jobs:
...
-        uses: SonarSource/jfrog-setup-wrapper@eb712d76540e5d1f3756303f30387657fb204e52  # tag=2.4.1-1
-        with:
-          jfrogAccessToken: ${{ fromJSON(steps.secrets.outputs.vault).promoter_access_token }}
       - name: Promote npm package
+        env:
+          PROMOTER_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).promoter_access_token }}
         run: |
-          jfrog rt bpr --status it-passed eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-builds
-          jfrog rt bpr --status released eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-releases
+          jfrog rt bpr --access-token $PROMOTER_TOKEN https://repox.jfrog.io/artifactory --status it-passed eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-builds
```
[the remark this answers is not in the document]
Comment (tomverin) [reply .github/workflows/release.yml:48]: indeed

[the remark this answers is not in the document]
Comment (saberduck) [reply .github/workflows/release.yml:49]: you are still not using the variable?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -41,11 +41,10 @@ jobs:
...
-          jfrogAccessToken: ${{ fromJSON(steps.secrets.outputs.vault).promoter_access_token }}
       - name: Promote npm package
+        env:
+          PROMOTER_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).promoter_access_token }}
+          REPOX_URL: ${{ fromJSON(steps.secrets.outputs.vault).repox_url }}
         run: |
-          jfrog rt bpr --status it-passed eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-builds
-          jfrog rt bpr --status released eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-releases
+          jfrog rt bpr --access-token $PROMOTER_TOKEN https://repox.jfrog.io/artifactory --status it-passed eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-builds
+          jfrog rt bpr --access-token $PROMOTER_TOKEN https://repox.jfrog.io/artifactory --status released eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-releases
```
Comment (saberduck) [inline .github/workflows/release.yml:50]:
```suggestion
          jfrog rt bpr --access-token $PROMOTER_TOKEN $REPOX_URL --status it-passed eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-builds
          jfrog rt bpr --access-token $PROMOTER_TOKEN $REPOX_URL --status released eslint-plugin-sonarjs ${{ github.event.release.tag_name }} sonarsource-npm-public-releases
```

**Review by saberduck — COMMENTED** [review]
@tomverin pls see my suggestion

**Review by saberduck — APPROVED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (julien-carsique-sonarsource) [reply .github/workflows/release.yml:50]: According to the doc, there should be no REPOX_URL in the parameters

**Pull request merged**